product-features/liquidity-pool/README.md

```diff
@@ -1,17 +1,60 @@
 # 🐋 Liquidity Pool
 
-## Liquidity Pool
+## Getting Started
 
-You can become a liquidity provider by submitting two tokens to the liquidity pool to begin earning a share of the trading fees.
```

Comment:
You can become a liquidity provider on ALEX by submitting two tokens to the liquidity pool to begin earning a share of the pool trading fees.